routes/user.js

```diff
@@ -1,18 +1,21 @@
-const express = require('express')
-
-const idValidation = require('~/middlewares/idValidation')
-const asyncWrapper = require('~/middlewares/asyncWrapper')
-const userController = require('~/controllers/user')
-const reviewRouter = require('~/routes/review')
-
-const router = express.Router()
-
-router.param('id', idValidation)
-
-router.use('/:id/:role/reviews', reviewRouter)
-
-router.get('/', asyncWrapper(userController.getUsers))
-router.get('/:id/:role', asyncWrapper(userController.getOneUser))
-router.delete('/:id', asyncWrapper(userController.deleteUser))
-
-module.exports = router
+const express = require('express')
+
+const idValidation = require('~/middlewares/idValidation')
+const asyncWrapper = require('~/middlewares/asyncWrapper')
+const userController = require('~/controllers/user')
+const reviewRouter = require('~/routes/review')
+const { restrictTo } = require('~/middlewares/auth')
+const { roles } = require('~/consts/auth')

```

Comment:
const { roles: { ADMIN } } = require('~/consts/auth')

Reply:
i have refactored it